README.md

```diff
@@ -2,39 +2,60 @@
 
 # OCaml - Continuous benchmarks
 
-Prototype for running predictable, IO-bound benchmarks in an ocurrent pipeline. This is *work in progress*.
-If you want to be on the allowlist for running benchmarks for your repository, please contact @gs0510, or you can open an issue.
+Prototype for running predictable, IO-bound benchmarks in an ocurrent pipeline. This is *work in progress*. If you want to be on the allowlist for running benchmarks for your repository, please contact @gs0510, ask around on [slack](https://example.com)<!--- FIXME: add real slack link if publishing it isn't an issue -->, or [open an issue](https://github.com/ocurrent/current-bench/issues/new).
```

Comment:
I searched GitHub and ocaml.org and a few places, and it seems like the Slack link is not published anywhere, and I guess we might want to keep it that way. We can change the contact info to point to you, Shakthi or me.

Reply:
I added our 3 github tags :)